src/services/branch-manager.js

```diff
@@ -6,8 +16,53 @@ async function deleteBranch() {
   // FIXME: Implement deleteBranch function
 }
 
-async function createBranch() {
-  // FIXME: Implement createBranch function
+async function createBranch(branchName) {
+  const authToken = authenticator.getAuthToken();
+
+  await agent
+    .post(`${serverHost()}/api/branches`)
+    .set('Authorization', `Bearer ${authToken}`)
+    .set('forest-secret-key', `${process.env.FOREST_ENV_SECRET}`)
```

Comment:
Since we'll have to add the `--project` anytime soon, maybe the `envSecret` should be passed to the function instead of using `process.env` here ?